package.json

```diff
@@ -42,7 +41,9 @@
     "supertest": "^4.0.2"
   },
   "jest": {
-    "globalSetup": "<rootDir>/test/jest.setup.js"
+    "setupFiles": [
```

Comment:
You might be able to replace this with `"testEnvironment": "testing"`.

Reply:
Hmm didn't work -- I think it's a different kind of test environment: https://jest-bot.github.io/jest/docs/configuration.html#testenvironment-string